mffpy/header_block/header_block.py

```diff
@@ -187,5 +160,5 @@ def encode_rate_depth(rate: int, depth: int) -> int:
         return (rate << 8) + depth
 
     @staticmethod
-    def compute_byte_size(num_channels: int) -> int:
-        return 4 * (4 + 2 * num_channels) + len(PADDING)
+    def compute_byte_size(num_channels: int, optional) -> int:
+        return 4 * (5 + 2 * num_channels) + optional.byte_size
```

Comment:
Note that the byte length of the optional header is not part of it.  Therefore we increment the regular header length by one byte (5 -> 4)

Reply:
Could we include a comment on this method about this calculated value?